handlers/bot_handler.py

```diff
@@ -54,14 +54,36 @@ def execute(cls, slack_wrapper, args, timestamp, channel_id, user_id, user_is_ad
             raise InvalidCommand("Sorry, couldn't retrieve the git information for the bot...")
 
 
+class InviteCommand(Command):
+    """
+    Invite a list of members to the current channel, ignores members already
+    present.
+    """
+
+    @classmethod
+    def execute(cls, slack_wrapper, args, timestamp, channel_id, user_id, user_is_admin):
+        try:
+            current_members = slack_wrapper.get_channel_members(channel_id)
+            # strip uid formatting
+            invited_users = [user.strip("<>@") for user in args]
+            # remove already present members
+            invited_users = [user for user in invited_users if user not in current_members]
+            for member in invited_users:
+                if not slack_wrapper.invite_user(member, channel_id)["ok"]:
+                    raise SlackClientError
```

Comment:
Why do try-raise-catch-raise? You could just raise InvalidCommand directly.

Also instead of raising the exception as soon as one user couldn't be invited, you might just want to continue inviting and populate a list with users, that weren't possible to invite. 

After the command finished you could raise an InvalidCommand stating which users couldn't be invited successfully. This way if there's a typo in one username the others might still be invited.

Reply:
Usernames are resolved into uids before the message is even received through the api, but this approach is more robust and has been added